.circleci/config.yml

```diff
@@ -223,7 +223,8 @@ jobs:
             - girder_env
 
   py2_integrationTests:
-    machine: true
+    machine:
+      image: ubuntu-1604:201903-01
```

Comment:
Note, this makes the integration CI tests now run under Ubuntu 16.04, instead of 14.04.

CircleCI does not yet offer a machine image with 18.04: https://circleci.com/docs/2.0/configuration-reference/#available-machine-images